rest-service/manager_rest/persistent_storage.py

```diff
@@ -11,6 +14,53 @@
 from manager_rest.rest.rest_utils import make_streaming_response
 
 
+LAST_MODIFIED_FMT = '%Y-%m-%dT%H:%M:%S.%f%z'
+
+
+@dataclass(slots=True)
+class FileInfo:
+    """FileInfo is a class used to represent information about a file"""
+    filepath: str
+    checksum: str
```

Comment:
we don't actually need the checksum, I think. Everything works off of just mtime